Source/OxyPlot/Axes/Rendering/MagnitudeAxisRenderer.cs

```diff
@@ -40,6 +40,9 @@ public override void Render(Axis axis, int pass)
 
             var angleAxis = this.Plot.DefaultAngleAxis;
 
+            angleAxis.AxisLineArea = new OxyRect(0, 0, 0, 0);
```

Comment:
Worth adding a null IShape?

Reply:
I'm currently struggling how to do this. Could you give me a hint?

Reply:
I just mean a boring implementation of `IShape` where `bool Contains(double x, double y);` always returns `false`; call it `EmptyShape` or `NullShape` or something: should be nicer than using zero-dimension Rects.

Reply:
ok, done

Reply:
Incidentally, I'm thinking `IOxyRegion` might be a better name than `IShape`, but that's easy to change once the everything else is nailed down. Thoughts welcome.

Reply:
Sounds good... Sold